root/static/scripts/edit/URLCleanup.js

```diff
@@ -4312,10 +4312,10 @@ const CLEANUPS: CleanupEntries = {
     match: [/^(https?:\/\/)?([^/]+\.)?offiziellecharts\.de\//i],
     restrict: [LINK_TYPES.otherdatabases],
     clean(url) {
-      return url.replace(/^(?:https?:\/\/)?(?:[^/]+\.)?offiziellecharts\.de\/([^/?#]+).*$/, 'https://www.offiziellecharts.de/$1');
+      return url.replace(/^(?:https?:\/\/)?(?:[^/]+\.)?offiziellecharts\.de\/charts\/([^/?#]+).*$/, 'https://www.offiziellecharts.de/charts/$1');
```

Comment:
Both old format (without `/charts`) and new format (with `/charts`) should probably be handled.

Whether you prefer to keep the old format or switch to the new one is up to you.

We didn’t receive any mailed answer from them so neither of these formats are permalinks.

Reply:
Ok, made the change to add `/charts` when missing (since that's where they redirect). Also added a quick improvement in a second commit to support `compilation` for RGs (and a test, which also tests that `/charts` is added).